src/source/plantrip.cpp

```diff
@@ -12,3 +12,38 @@ PlanTrip::~PlanTrip()
 {
     delete ui;
 }
+
+/**
+ * @brief PlanTrip::propagateStadiumTable
+ * This method takes a database object that is sqlite database and retrieve the
+ * stadiums / teams within the database. It will only propagate the stadium name
+ * and team name into the table view.
+ * @param db
+ * database object that inherits the qt sqlite database.
+ */
+void PlanTrip::propagateStadiumTable(Database *db)
+{
+    QSqlTableModel* model = new QSqlTableModel(this,db->database());
+
+    model->setTable("stadiums");
+    model->select();
+    model->setHeaderData(1, Qt::Horizontal, tr("Team Name"));
+    model->setHeaderData(2, Qt::Horizontal, tr("Stadium Name"));
+
+    if(ui->planTrip_tableView_stadiums == NULL)
+    {
+        ui->planTrip_tableView_stadiums = new QTableView;
+    }
+    ui->planTrip_tableView_stadiums->setModel(model);
```

Comment:
these should be within some initialize function... magic numbers should be replaced with appropriate enum values from the stadiumTableModel